packit_service/worker/events/pagure.py

```diff
@@ -90,11 +90,18 @@ def get_dict(self, default_dict: Optional[Dict] = None) -> dict:
         return result
 
     def get_base_project(self) -> GitProject:
+        if isinstance(self, PullRequestCommentPagureEvent):
```

Comment:
personal preference: I would use ternary operator here

Reply:
maybe I am missing something, but since this is a method of `PullRequestCommentPagureEvent`, can the `else` block happen?

Reply:
Yes you are right! I was thinking of being in a more general PagureEvent...